bin/smart_proxy_dynflow_core

```diff
@@ -3,5 +3,22 @@ require 'dynflow'
 require 'rack'
 require 'smart_proxy_dynflow_core/launcher'
 require 'yaml'
+require 'optparse'
```

Comment:
`optparse` is in ruby's stdlib so we won't need to package it. `clamp` felt like an overkill for this use case.